bazel_tools/pom_template.xml

```diff
@@ -20,15 +20,6 @@
     </license>
   </licenses>
 
-  <developers>
-    <developer>
-      <name>Digital Asset SDK Feedback</name>
-      <email>sdk-feedback@digitalasset.com</email>
-      <organization>Digital Asset (Switzerland) GmbH</organization>
-      <organizationUrl>http://www.daml.com</organizationUrl>
-    </developer>
-  </developers>
```

Comment:
Don't we want a reference to DAML Connect here now?

Reply:
@bame-da suggested to kill it completely and I agree that the pom file in our example sis not the right place for support.

Reply:
That makes sense. Thank you.